metadata.yaml

```diff
@@ -0,0 +1,8 @@
+service:
+  full_name:
+  short_name:
+  link:
+  documentation:
```

Comment:
Can we fill those with working URLs? e.g https://github.com/aws-controllers-k8s/dynamodb-controller/blob/main/metadata.yaml

Reply:
updated metadata.yaml